sklearn/hpsearch/kubernetes_helper.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+from kubernetes import client, config
+
+# brew install python to get 2.7.13 which has updated openssl
+# check openssl version with python -c "import ssl; print ssl.OPENSSL_VERSION"
+# mkvirtualenv -p /usr/local/Cellar/python/2.7.13_1/bin/python2 hpsearch 
```

Comment:
What's the context for these 3 steps? (Are they important to the guide?)

Reply:
removed.